src/Cake.Coverlet/CoverletOutputFormat.cs

```diff
@@ -1,28 +1,32 @@
 ﻿using System;
 
-namespace Cake.Coverlet

```

Comment:
This file had a mix of different line endings. VS Code made them all consistent Unix line endings - if you don't like that change I can undo that part.